package ExceptionError;

/*
 * Exception : Exception is unexpected event which occurs at runtime It may occur when user giving input to dynamic varaible 
 * or by Database engine while executing execution in jdbc query or may occurs during Network issue while connecting to other system 
 * causes abnormal termination to the application .
 * 
 * 
 * we need to handle Exceptions at runtime for smooth termination of program .
 * Predefined class to handle predefined Exceptions 
 *********************************************RUNTIME EXCEPTION *********************************************
 *Arithmetic Exception
 *ClassCastException
 *IndexOutOfBondException ---(ArrayIndexOutOfBondException StringIndexOutOfBondException )
 *NullPointerException 
 */
public class secondExceptionProgram {

	public static void main(String[] args) {
		
		
		
		
	}

}
